content/influxdb/v1.8/tools/influx_inspect.md

```diff
@@ -452,6 +454,78 @@ The flag to report exact cardinality counts instead of estimates.
 Default value is `false`.
 Note: This can use a lot of memory.
 
+### `report-disk`
+
+Use the `report-disk` command to review disk usage by shards and measurements for TSM files in a specified directory. Useful for determining disk usage for capacity planning and identifying which measurements or shards are using the most space.
```

Comment:
What is the unit for the reported size? bytes? or ??

Reply:
Thanks, Tim. Looks like bytes in code. @lesam can you confirm?